tests/examples/redirects.html

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <!-- <link href="http://httpbin.org/redirect-to?url=http%3A//minimalcss.local/finally.css" rel=stylesheet> -->
+    <link href="http://httpbin.org/redirect-to?url=http%3A//httpbin.org/redirect-to%3Furl%3Dhttp%253A//minimalcss.local/finally.css" rel=stylesheet>
```

Comment:
I was having problems with content-types with my `http-server` so I set up a little nginx with a local domain. 
I'm obviously going to rewrite this using `fastify` for the tests.